packages/aws-cdk/lib/api/hotswap-deployments.ts

```diff
@@ -59,13 +66,58 @@ export async function tryHotswapDeployment(
 
 async function findAllHotswappableChanges(
   stackChanges: cfn_diff.TemplateDiff, evaluateCfnTemplate: EvaluateCloudFormationTemplate,
+  sdk: ISDK, rootStackArtifact: cxapi.CloudFormationStackArtifact, nestedStackNames: {[key: string]: any}, hotswappableResources?: HotswapOperation[],
```

Comment:
Why are `nestedStacknames`'s value type is `any`? Why cannot we narrow it down further?